src/config.rs

```diff
@@ -20,12 +20,20 @@ pub struct Config {
     pub message_dur: Duration,
     /// Whether to display line numbers.
     pub show_line_num: bool,
+    /// Whether to display nonprintable characters.
+    pub non_printable: bool,
```

Comment:
Use a shorter name here so that the default function fits into one line.

Reply:
I think the formatting doesn't care about the length of the default function when there are too many arguments. I tried to shorten all names and it still moves them on a new line.